layouts/partials/contact.html

```diff
@@ -15,7 +15,11 @@ <h3>{{ i18n "contactForm" }}</h3>
 
                 <div id="contact-message"></div>
 
-                <form class="contact-form" method="post" action="https://formspree.io/{{ .Site.Params.email }}">
+                {{ if .Site.Params.contact_form_ajax }}
```

Comment:
You could squeeze this into:

```html
<form class="contact-form-{{ with .Site.Params.contact_form_ajax }}ajax{{ else }}post{{ end }}" method="post" action="https://formspree.io/{{ .Site.Params.email }}">
```

Reply:
I agree

Reply:
Thanks for the tip.

I updated the pull request as per your request.